src/EvoSC/InternalModules.cs

```diff
@@ -24,6 +25,8 @@ public static class InternalModules
         typeof(SetNameModule),
         typeof(FastestCpModule),
         typeof(CurrentMapModule),
+        typeof(FastestCpModule),

```

Comment:
Duplication issue. Seems like the rebase left some artifacts.

Reply:
Fixed